src/main.rs

```diff
@@ -59,8 +60,18 @@ fn main() {
     };
 
     setup_logger(log_target, log_level);
-
-    cli_config.load_config_file_values();
+    color_eyre::install().expect("Coundn't initialize error reporting");
+
+    cli_config
+        .load_config_file_values()
+        .wrap_err("could not load the config file")
+        .with_section(|| {
+            concat!(
+                "the config format should be valid TOML\n",
+                "we recently changed the config format, check the README for more info"
```

Comment:
Looks fine! The only change I'd come up with would be to add a link to the certain github issue(s) here. That would make it easier to troubleshoot the known config issue(s). What do you think?

Reply:
this is the way color-eyre provides a stack of context, from the actual error to what that error means in the application. I can see how it looks like 2 errors, ill look if the lib supports some customization otherwise maybe we should remove it

Reply:
mm i cant seem to change the `Error:` to something like `Caused by:`, but removing the line `could not load the config file` would make it less obvious that the error came from loading the config file. So i think this is the best for now

Reply:
> The only change I'd come up with would be to add a link to the certain github issue(s) here

there is even some support for that (https://docs.rs/color-eyre/0.5.10/color_eyre/config/struct.HookBuilder.html#method.issue_url).